recaf-ui/src/main/java/me/coley/recaf/ui/plugin/MarketplaceTab.java

```diff
@@ -0,0 +1,28 @@
+package me.coley.recaf.ui.plugin;
+
+import javafx.scene.control.Tab;
+import me.coley.recaf.ui.plugin.item.MarketplacePluginItem;
+import me.coley.recaf.ui.plugin.repository.CommonPluginRepository;
+import me.coley.recaf.ui.plugin.repository.PluginRepository;
+import me.coley.recaf.util.logging.Logging;
+import org.slf4j.Logger;
+
+import java.util.List;
+
+/**
+ * @author xtherk
+ */
+public class MarketplaceTab extends Tab {
```

Comment:
While modeled after the IntelliJ marketplace layout, we'll probably want to call it something else _(I have no intention of having paid plugins, so 'market' seems to suggest the wrong idea)_. I suck at naming things though. Can ignore for now, just my own thoughts.

Reply:
Maybe change to "Remote"  :)

Reply:
Temporary changed to "Marketplace Free" :)